images/kubevirt-userguide/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.fedoraproject.org/fedora:latest
+
+RUN cd /
```

Comment:
The cd won't affect following commands. If you want to change the directory where you execute, use WORKDIR